_includes/documentation-links-band.html

```diff
@@ -58,6 +58,9 @@ <h4><i class="fas fa-code"></i> Developers</h4>
         <li>
           <a href="{{site.data.ispn.stable.docs.xsite}}" target="_blank">Cross-site replication</a>
         </li>
+        <li>
+          <a href="{{site.data.ispn.stable.docs.resp}}" target="_blank">Using Redis clients</a>
```

Comment:
Hii  @karesti Thanks for this! I just noticed the title is not on the website. 
WDYT about?
```suggestion
          <a href="{{site.data.ispn.stable.docs.resp}}" target="_blank">Using the RESP protocol endpoint</a>
```

Reply:
Hi @dvagnero I think it's better using redis clients since RESP protocol endpoint is too low level. From a user perspective it means using a Redis Client in the code but  having Infinispan instead